site/docs/plugins/parse-mode.md

```diff
@@ -43,10 +43,10 @@ import { hydrateReply, parseMode } from "parse-mode";
 
 const bot = new Bot("");
 
-// Install familiar reply variants to ctx
+// Use the plugin.
```

Comment:
In all cases where we are adding a plugin which is not the main focus of the page it would be nice to include a little information instead of just "Use this plugin".

Currently, the change in this line, for example, makes it harder to understand (or guess) what `hydrateReply` might do.